### PR TITLE
Update `KeyboardEvent.code` values on Linux and macOS

### PR DESCRIPTION
**Pull request by CryZe**
#### Summary
A lot of these values are inaccurate, not up to date, or just plain wrong.

This is a follow up to #9562 which updated them on Windows.

#### Motivation
Because they are wrong.

#### Supporting details
https://github.com/chromium/chromium/blob/master/ui/events/keycodes/dom/dom_code_data.inc
https://github.com/mozilla/gecko-dev/blob/master/widget/NativeKeyToDOMCodeName.h

#### Related issues
OSLeft / OSRight: https://bugzilla.mozilla.org/show_bug.cgi?id=1595863
VolumeDown / VolumeMute / VolumeUp: https://bugzilla.mozilla.org/show_bug.cgi?id=1272579

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by CryZe** [comment]
There's also some non-standardized Chromium only values. Not sure if we want to list them. They almost even seem accidental to some degree.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/keyboardevent/code/code_values/index.md [hunk 1]
```diff
@@ -1059,7 +1059,6 @@ In the cells,
   when using it;
 - "(⚠️ Same string for `0xab`)" means that you cannot distinguished this key with the one matching `0xab`;
 - "(⚠️ No events fired actually)" means that even if technically you have a specific string for this code, no event will be dispatched;
-- "(⚠️ Different string on xyz)" means that the same key will have a different name on browser xyz.
```
[the remark this answers is not in the document]
Comment (CryZe) [reply files/en-us/web/api/keyboardevent/code/code_values/index.md:1062]: However looking at the currently released page again, `(⚠️ Not the same on xyz)` is only used in the Windows table and `(⚠️ Different string on xyz)` is only used in the others. They seem to mean the exact same thing, so the unification seems warranted. I can unify it to be the latter though if you prefer that.

**Comment by Elchi3** [comment]
Maybe @masayuki-nakano could help reviewing this?

**Comment by masayuki-nakano** [comment]
Sorry, I don't have much time to check the big diff and I don't remember the history of name changes in the spec and in Gecko, so for checking this, I need to investigate a lot for taking back the memory.

**Comment by masayuki-nakano** [comment]
(But feel free to ask me for specific issue for a key.)

File: files/en-us/web/api/keyboardevent/code/code_values/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/keyboardevent/code/code_values/index.md:1062]: Sorry. The earlier reference is not visible by default when I'm reviewing. I see it now.

File: files/en-us/web/api/keyboardevent/code/code_values/index.md [hunk 2]
```diff
@@ -2513,12 +2523,12 @@ In the cells, "(❌ Missing)" means that this code value cannot be detected on t
     <tr>
       <th scope="row"><code>0x00BB</code></th>
       <td><code>"Unidentified"</code> (❌ Missing)</td>
-      <td><code>"NumpadParenLeft"</code></td>
+      <td><code>"NumpadParenLeft"</code> (⚠️ Not the same on Chromium)</td>
```
Comment (wbamberg) [inline files/en-us/web/api/keyboardevent/code/code_values/index.md:2526]:
```suggestion
      <td><code>"NumpadParenLeft"</code> (⚠️ Not the same on Gecko)</td>
```

File: files/en-us/web/api/keyboardevent/code/code_values/index.md [hunk 3]
```diff
@@ -2513,12 +2523,12 @@ In the cells, "(❌ Missing)" means that this code value cannot be detected on t
...
     </tr>
     <tr>
       <th scope="row"><code>0x00BC</code></th>
       <td><code>"Unidentified"</code> (❌ Missing)</td>
-      <td><code>"NumpadParenRight"</code></td>
+      <td><code>"NumpadParenRight"</code> (⚠️ Not the same on Chromium)</td>
```
Comment (wbamberg) [inline files/en-us/web/api/keyboardevent/code/code_values/index.md:2531]:
```suggestion
      <td><code>"NumpadParenRight"</code> (⚠️ Not the same on Gecko)</td>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
This is an impressive update @CryZe !

I've done my best to compare the values you've changed here with the ones in the sources you've given, and they look good. Seems like there are a couple of places where you say "Chromium" rather than "Gecko".

I agree that the whole "Not the same on XYZ" is clunky, but you didn't invent it in this PR and the PR seems to correct lots of errors, so I think it should be fine to merge with the 2 changes I suggested.

File: files/en-us/web/api/keyboardevent/code/code_values/index.md [hunk 4]
```diff
@@ -1417,35 +1416,26 @@ In the cells,
     </tr>
     <tr>
       <th scope="row"><code>kVK_VolumeUp (0x48)</code></th>
+      <td><code>"VolumeUp"</code> (⚠️ Not the same on Chromium)</td>
```
[the remark this answers is not in the document]
Comment (wbamberg) [reply files/en-us/web/api/keyboardevent/code/code_values/index.md:1419]: The specifics are, I think, that the value in the Gecko cell ("VolumeUp") is different from the value for the Chromium cell ("AudioVolumeUp").

It is not an ideal way to communicate the difference, but it seems clear enough what it is saying, and the OP didn't invent this scheme, so I don't think it should block the PR, which seems to fix some real issues in the page.

**Review by wbamberg — APPROVED** [review]
👍 thanks @CryZe !

@jpmedley , I think we should merge this PR, and if needed deal with the "Not the same on ..." stuff in another PR, since this PR did not introduce that system and seems to add lots of fixes to the content. What do you think?

**Review by jpmedley — APPROVED** [review]

**Pull request merged**